recipe/meta.yaml

```diff
@@ -18,7 +18,9 @@ source:
       - patches/tklib_build.patch
       - patches/clibffi.patch  # [osx]
       - patches/darwin.patch  # [osx]
+      # Patches to be removed in the next version
       - patches/distutils.patch
```

Comment:
This package is not upstreamed. I can send it if you think it's okay to have that patch upstream

Reply:
Done in [pypy/py3.7 HEAD](https://foss.heptapod.net/pypy/pypy/-/commit/f571351456ad27d1dbb0e7325109a0e4fd70347a), thanks.